checker/build.gradle

```diff
@@ -1085,6 +1093,39 @@ task wpiPlumeLibTest(group: 'Verification') {
     }
 }
 
+// Unzip JSpecify conformance tests using the artifact
+task unzipConformanceTestSuite(group: 'Verification') {
+    description 'Unzips the conformance test suite into the build directory.'
+    copy {
+        from zipTree(configurations.conformanceTestSuite.singleFile)
+        into "${buildDir}/conformance-test-suite"
+    }
+}
+
+// Run JSpecify conformance tests
+task conformanceTests(type: Test, group: 'Verification') {
+    description 'Runs the conformance tests.'
+    include '**/*ConformanceTest.class'
+
+    shouldRunAfter test
+
+    environment "JSPECIFY_CONFORMANCE_TEST_MODE", "WRITE"
+
+    // Conformance tests
+    inputs.files(unzipConformanceTestSuite)
+    inputs.files("${projectDir}/report/ConformanceTest-report.txt")
+
+    doFirst {
+        systemProperties([
+            "ConformanceTest.inputs": "${buildDir}/conformance-test-suite/assertions/org/jspecify/conformance/tests",
+            "ConformanceTest.report": "${projectDir}/report/ConformanceTest-report.txt",
```

Comment:
How about putting the file here: `"${projectDir}/src/test/resources/jspecify-conformance-test-report.txt"`?